examples/browser/wdio.conf.js

```diff
@@ -1,3 +1,11 @@
+// @ts-check 
+const http = require('http');
+const path = require('path');
+
+const theiaConfig = require(path.resolve(__dirname, '.yo-rc.json'))['generator-theia'];
```

Comment:
I would prefer to start the server with `src-gen/backend/main` and then use its address and port to configure wdio. Unfortunately, it is not possible. As another solution, we can allow passing host and port to `src-gen/backend/main` to override defaults from the generator.